### PR TITLE
fix some error in the code of  `setup orchard core cms` page, add som…

### PR DESCRIPTION
**Pull request by mildronize**
Please review, thank you very much 😎

### CODE REVIEW

File: content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md [hunk 1]
```diff
@@ -25,9 +25,11 @@ sudo apt-get update; \
 dotnet --list-sdks
 ```
 - It should return `5.0.202 [/usr/share/dotnet/sdk]` or a newer version of .NET 5.
-- Then, use the following command to install a new project template.
+
+# Install Orchard Template
+- Then, use the following command to install a new project template (In this case, it'll use [nightly build](https://github.com/OrchardCMS/OrchardCore/#build-status) for OrchardCore).
 ```sh
-dotnet new -i OrchardCore.ProjectTemplates::1.0.0-rc2-* --nuget-source https://nuget.cloudsmith.io/orchardcore/preview/v3/index.json
+dotnet new -i "OrchardCore.ProjectTemplates::1.0.0-rc2-*" --nuget-source https://nuget.cloudsmith.io/orchardcore/preview/v3/index.json
```
Comment (aaronamm) [inline content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md:32]: Do we really need quotes for a  template name? I think having quotes is safe but just curiosity, if without it still work.

File: content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md [hunk 2]
```diff
@@ -107,3 +109,23 @@ dotnet run
...
+--------------------------------------------  --------------  ----------  ----------------------
+Orchard Core Cms Module                       ocmodulecms     [C#]        Web/Orchard Core/CMS
+Orchard Core Cms Web App                      occms           [C#]        Web/Orchard Core/CMS
+Orchard Core Theme                            octheme         [C#]        Web/Orchard Core/CMS
+Orchard Core Mvc Module                       ocmodulemvc     [C#]        Web/Orchard Core/Mvc
+Orchard Core Mvc Web App                      ocmvc           [C#]        Web/Orchard Core/Mvc
+```
+
+For exmaple:
+Orchard Core Cms Web App:
```
Comment (aaronamm) [inline content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md]: Could you please have a shot description to explain what "dotnet new occms" does?
For example,
`Orchard Core Cms Web App:` to create a new Orchard Core CMS website project.

File: content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md (hunk 1 above)
Comment (mildronize) [reply content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md:32]: Without quote, it'll get error on WSL2 ;)

File: content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md (hunk 2 above)
Comment (mildronize) [reply content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md]: Sure

File: content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md (hunk 1 above)
Comment (mildronize) [reply content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md:32]: It got error, because I use `zsh` Lol

Comment (mildronize) [reply content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md:32]: I think with a quote, it's safer.

Comment (aaronamm) [reply content/web-frameworks/orchard-core-cms/setup-orchard-core-cms.md:32]: Amazing, I think it use it before and don't have any issue.
Anyway, it is safer to have quote. Thanks.

**Review by aaronamm — APPROVED** [review]

**Pull request merged**